### PR TITLE
Add readthedocs folder

### PR DESCRIPTION
**Pull request by dsariel**
1. Created doc folder with initial structure
2. Moved installation and usage instructions from README.md to docs
3. added docs env to tox.ini

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,6 @@
-# CIbyl
+CIbyl
```
Comment (cescgina) [inline README.md:1]: This is a markdown file but with rst syntax, will github be smart enough to parse it correctly? I just check and the link is not correctly shown. We can rename the file to rst or use markdown for it. Both would be fine IMO

File: README.md [hunk 2]
```diff
@@ -1,44 +1,6 @@
...
-## Usage
-
-To list existing environments and their systems, run `cibyl`
-
-## How to Contribute?
-
-Contributions are done via GitHub Pull Requests
+Official Documentation
+----------------------
+For more information please read our `Documentation <http://cibyl.readthedocs.org>`_
```
Comment (cescgina) [inline README.md:6]: In addition to the link to the docs, I  would keep the info that we had before as a quick glance for the user. I usually find  the github of a project before its docs and I think it's nice to be able to get some vague idea of the project from the readme

**Review by cescgina — CHANGES_REQUESTED** [review]
Minor issues with the README, but overall looks great

File: README.md (hunk 2 above)
Comment (bregman-arie) [reply README.md:6]: 100% on what Joan wrote.
I really like to have quick glance in README.md on what the project is about + quick start (installation, perhaps basic usage, ...)

File: docs/Makefile [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    = -W
+SPHINXBUILD   = sphinx-build
+SPHINXPROJ    = infrared
```
Comment (jsanemet) [inline docs/Makefile:7]: I think we can remove this variable, it does not belong to our project.

File: docs/Makefile [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+SOURCEDIR     = source
```
Comment (jsanemet) [inline docs/Makefile:8]: On our case, sources are stored on the 'cibyl' folder.

File: docs/source/conf.py [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    'reno.sphinxext']
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
```
Comment (jsanemet) [inline docs/source/conf.py:40]: If we are to use '.rst' files, then I am with @cescgina in changing to README.rst

**Review by jsanemet — CHANGES_REQUESTED** [review]

File: README.rst [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+Command line interface for querying CI environments
+
+Installation
+************
+
+```
+cd cibyl 
+virtualenv ~/.cibyl_venv && source ~/.cibyl_venv/bin/activate
+sudo pip install .
```
Comment (IgnacioPro) [inline README.rst:12]: Should we add the "pip install -r requirements.txt" maybe?

**Review by IgnacioPro — CHANGES_REQUESTED** [review]

File: README.rst (hunk 6 above)
Comment (bregman-arie) [reply README.rst:11]: Isn't it installed when running "sudo pip install ."? not sure

Comment (IgnacioPro) [reply README.rst:11]: Just tried and no.

Comment (bregman-arie) [reply README.rst]: changed to "sudo pip install cibyl". When installing from PyPi, it will install the requirements as well

**Comment by jsanemet** [comment]
Please, be aware of https://github.com/rhos-infra/cibyl/pull/77

**Comment by bregman-arie** [comment]
> Please, be aware of #77

Changed the path

File: docs/source/usage.rst [hunk 7]
```diff
@@ -0,0 +1,10 @@
+Usage
+-----
+
+To list existing environments and their systems, run `cibyl`
+
+To list all job, run `cibyl --jobs`
```
Comment (cescgina) [inline docs/source/usage.rst:6]: s/job/jobs

**Review by cescgina — APPROVED** [review]
Looks good, I just found a typo

**Pull request merged**